### DISCUSSION TITLE
Cannot get Meteor + Quasar  to work!

### DISCUSSION THREAD

**Discussion by yuniit** [quasarframework/quasar#13003] [General - Components / Directives / etc]
Hi all,
Can anyone help me to make meteor work with Quasar?

meteor needs `fourseven:scss`  to be able to use sass, scss 
but after importing `'quasar/src/css/index.sass'` in `app.js` I got this error:

```js
/home/rabbit/.meteor/packages/meteor-tool/.2.7.0.1gne3m2.dzd0l++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/meteor-promise/promise_server.js:218
      throw error;
      ^
{
  message: `Scss compiler error: Error: Invalid CSS after "..., $replace: '')": expected "{", was "$index: str-index($"\n` +
    '        on line 1 of node_modules/quasar/src/css/helpers/string.sass\n' +
    '        from line 1 of {}/node_modules/quasar/src/css/index.sass\n' +
    ">> str-replace($string, $search, $replace: '')\n" +
    '\n' +
    '   ------------------------------------------^\n' +
    '\n',
  info: {
    file: '/node_modules/quasar/src/css/index.sass',
    line: undefined,
    column: undefined,
    func: undefined
  },
  stack: 'undefined\n' +
    ' => awaited here:\n' +
    '    at Promise.await (/home/rabbit/.meteor/packages/meteor-tool/.2.7.0.1gne3m2.dzd0l++os.linux.x86_64+web.browser+web.browser.legacy+web.cordova/mt-os.linux.x86_64/dev_bundle/lib/node_modules/meteor-promise/promise_server.js:60:12)\n' +

....
... more
```

Vue: `^3.2.31`
Quasar: `2.6.2`

Here is my [meteor + quasar repo](https://github.com/yuniit/meteor-quasar/tree/main)

thank in advance!

**Comment by yusufkandemir** [comment]
> Note that due to a limitation in libsass, there is **no support** for **imports** with **indented syntax (sass)**. Indented syntax does work on the top-level. A version based on dart-sass is **in the works which should remove this limitation**.
> 
> Meteor 1.7 introduced a change in how node_modules are handled, **if you want to import sass from a node_module you need to symlink the package in your imports directory** (more information below)

https://github.com/Meteor-Community-Packages/meteor-scss#sass-for-meteor

These are limitations of `fourseven:css`, and they are clearly listed on their README file. They stated that another version is in the works, so you might want to wait for that. For any other help about this specific topic please seek help in the Meteor community as this is not a Quasar problem.

**Comment by ontwikkelfabriek** [comment]
@yuniit 

Out of experience I would highly recommend to use Quasar CLI with the Meteor Client Bundler package for creating Meteor ( headless, server only) + Quasar ( separate client app) applications.

Bootstrap project with Meteor backend + Quasar Frontend app with Meteor Client Bundler ( now with vue3 ):
https://github.com/alexandesigner/quasar-meteor

Meteor Client Bundler repo:
https://github.com/Urigo/meteor-client-bundler

**Comment by yuniit** [comment]
I might need to wait while using CDN style for now.
Thank
